db/migrate/20171010200935_create_portal_offering_activity_feedbacks.rb

```diff
@@ -0,0 +1,15 @@
+class CreatePortalOfferingActivityFeedbacks < ActiveRecord::Migration
+  def change
+    create_table :portal_offering_activity_feedbacks do |t|
+      t.boolean :enable_text_feedback, default: false
+      t.integer :max_score, default: 10
+      t.string  :score_type, default: "none"
+      t.references :activity, index: true, foreign_key: true
+      t.references :portal_offering, index: true, foreign_key: true
+      t.timestamps
+    end
+  end
+end
+
+
+# /Users/npaessel/lab/cc/portal/db/migrate/20171010200935_create_portal_offering_activity_feedbacks.rb
```

Comment:
this should be removed. :)

Reply:
Nice catch. My original intention was to sort by `created_at`.  Not sure what happened.  This seems easier and more obvious than maintaining another sort order.

Also not sure why / how the file path ended up in the migration(?) 

Thanks!